lua/cfc_time/server/storage_options/mysql.lua

```diff
@@ -207,9 +297,37 @@ function storage:GetTotalTime( steamID, callback )
     query:start()
 end
 
-function storage:CreateSession( callback, steamID, sessionStart, sessionEnd, duration )
-    local newSession = self:Prepare( "newSession", callback, steamID, sessionStart, sessionEnd, duration )
-    newSession:start()
+function storage:CreateSession( callback, steamID, sessionStart, sessionEnd, sessionDuration )
+    local maxDuration = self.MAX_SESSION_DURATION
+    local sessionsCount = math.ceil( maxDuration / sessionDuration )
```

Comment:
```suggestion
    local sessionsCount = math.ceil( sessionDuration / maxDuration )
```
Should be total time / interval time to get interval count